tests/dbus_service_mocks/udisks2.py

```diff
@@ -0,0 +1,41 @@
+"""Mock of base UDisks2 service."""
+
+from dbus_fast import Variant
+from dbus_fast.service import signal
+
+from .base import DBusServiceMock, dbus_method
+
+BUS_NAME = "org.freedesktop.UDisks2"
+
+
+def setup(object_path: str | None = None) -> DBusServiceMock:
+    """Create dbus mock object."""
+    return UDisks2()
+
+
+class UDisks2(DBusServiceMock):
+    """UDisks2 base object mock.
+
+    gdbus introspect --system --dest org.freedesktop.UDisks2 --object-path /org/freedesktop/UDisks2/Manager
```

Comment:
Tiny nit, just mocks a different path:
```suggestion
    gdbus introspect --system --dest org.freedesktop.UDisks2 --object-path /org/freedesktop/UDisks2
```